pkg/internal/checkup/checkup.go

```diff
@@ -247,14 +248,23 @@ func (c *Checkup) checkVersions(ctx context.Context) error {
 func (c *Checkup) checkGoldenImages(ctx context.Context, namespaces *corev1.NamespaceList, errStr *string) error {
 	log.Print("checkGoldenImages")
 
+	const defaultGoldenImagesNamespace = "openshift-virtualization-os-images"
 	var cs goldenImagesCheckState
-	for i := range namespaces.Items {
-		ns := namespaces.Items[i].Name
-		if err := c.checkDataImportCrons(ctx, ns, &cs); err != nil {
+
+	if ns, err := c.client.GetNamespace(ctx, defaultGoldenImagesNamespace); err == nil {
```

Comment:
question, dont you get in the namespaces list also "openshift-virtualization-os-images"? if so why do you need this commit?

Reply:
I want to prefer it over other namespaces if we have DataImportCrons there, but use DICs from the other namespaces if not. It can be written as a single loop but it will be a bit more clumsy.

Reply:
ah I see that inside checkDataImportCrons you eventually update some values so now I understand